Contribute/style.md

```diff
@@ -5,33 +5,41 @@ title: C Coding Conventions and Style Guide
 pre: Coding Conventions
 pre_link: conventions.html
 sub: '<p>
-  Formatting, style, and C language subset requirements, divided into
-  <a href="#general-c-code">general</a>, <a href="#user-level-code">user-level</a>,
-  <a href="#kernel-code">kernel code</a>, and
-  <a href="#verification-requirements">verification requirements</a>.
+
+  Formatting, style, and C language subset requirements, divided into <a
+  href="#general-c-code">general</a>, <a href="#user-level-code">user-level
+  code</a>, and <a href="#kernel-code">kernel code</a> (including specific
+  requirements for <a href="#verification-requirements">verified kernel
+  code</a>).
+
 </p>'
 
 ---
 
-These are conventions which we attempt to follow in all of our code.
-They do not apply to external libraries, where we stick with the
-existing conventions.
+We strive to follow the below conventions for all the code maintained by the
+seL4 Foundation in the <a href="https://github.com/seL4">seL4 GitHub org</a>.
+We make an exception for external libraries, where we refer to the existing
+conventions.
```

Comment:
```suggestion
We make an exception for external libraries, where we use their existing
conventions.
```

Reply:
I think I want to tweak this a bit more. Attempt and strive are too weak -- they are actual requirements, we just have not applied them to old code.